### DISCUSSION TITLE
Date field implied by file prefix?

### DISCUSSION THREAD

**Discussion by gerwitz** [estruyf/vscode-front-matter#490] [Q&A] [answered]
I have many content files with a traditional `YYYY-MM-DD-slug.md` filename, and no date field int he front matter. This seems a common pattern but I can't seem to configure FM to parse the filename for this implied publication date field. Am I missing something?

**Comment by estruyf** [comment, marked as answer]
Front Matter doesn't use the date prefix on the file, as it expects date field(s) in its content type.

**Reply by gerwitz** [reply]
Many SSGs discern dates from paths and/or filenames … is this something FM might someday adopt, or do you want it to be opinionated that files should contain their metadata explicitly?

(No judgement implied! I'm reluctant to mass-modify my content files, but am noticing signals that filesystem structure is not as respected as it once was.)

**Reply by estruyf** [reply]
I haven't thought about it yet. It might be good to ask others how they use it and see the outcome. If others would rely on this feature, I am more than happy to put it in the backlog.
